sonar-db/src/test/java/org/sonar/db/component/ComponentDbTester.java

```diff
@@ -83,8 +83,13 @@ public void insertComponents(ComponentDto... components) {
     db.commit();
   }
 
-  public void indexProjects() {
+  public void indexProjectsAndViews() {
     dbClient.componentIndexDao().indexProjects();
```

Comment:
I think it should be best to either : move the code of the DAO here (but it will still requires some maintenance) or to not use at all this method and use instead ResourceIndexDao.indexResource(), which is used in prod.